Jobs update retries (#6762)
scaleway/docs-content: pages/serverless-jobs/how-to/manage-job-retries.mdx

The retry option lets you define how many times a Serverless Job automatically retries on failure. A retry is triggered whenever a job run ends in a `failed` status, regardless of the exit code returned by the runtime.

- **Retried** — job runs in a `failed` status with a valid exit code
- **Not retried**:

>     - job runs in `failed` status **without exit_code** (null), eg:  platform mismatch or other infrastructure-side error

Reviewer: ```suggestion
    - job runs in `failed` status **without an exit_code** (null), e.g.,  platform mismatch or other infrastructure-side error
```

>     - job runs in `failed` status **without an exit_code** (null), e.g.,  platform mismatch or other infrastructure-side error

Reviewer: ```suggestion
    - job runs in `failed` status **without an exit_code** (null), e.g.,  platform mismatch or some other infrastructure-side error
```

>     - job runs in `failed` status **without an exit_code** (null), e.g.,  platform mismatch or some other infrastructure-side error

Reviewer: ```suggestion
    - job runs in `failed` status **without an exit_code** (null), e.g., platform mismatch or some other infrastructure-side error
```

>     — job runs in an `interrupted` status.

Reviewer: ```suggestion
    — job runs in an `interrupted` status
```
